intelmq/bots/experts/aggregate/expert.py

```diff
@@ -46,8 +51,14 @@ def cleanup(self):
                     event.add("extra.count", int(data['c']))
                     event.add("extra.time_end", data['l'])
                     self.send_message(event)
+                    counter_sent += 1
+                else:
+                    counter_ignored += 1
```

Comment:
Maybe this case here is "message gets discarde because threshold was not reached"?

Reply:
Yes, if threshold is not reached, message gets dropped.